src/rpc/server.cpp

```diff
@@ -258,6 +258,22 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
     return "Bitcoin server stopping";
 }
 
+UniValue uptime(const JSONRPCRequest& jsonRequest)
```

Comment:
I understand both sides so don't have a strong opinion. We want to have multiple JSON-RPC endpoints at some point, but it seems harmless if they all export `uptime`.

However: `server.cpp` should be independent of bitcoin, so if you plan on keeping it there, please remove the explicit reference to Bitcoin below in the documentation.